packages/logic-utils/test/utils/documentation-spec.ts

```diff
@@ -0,0 +1,252 @@
+          },
+          model
+        )
+      ).to.equal("Deprecated. \nThe Title\n");
+    });
+
+    it("replaces <br> tags with newline", () => {
+      expect(
+        getDeprecationPlainTextSnippet(
+          undefined,
+          {
+            isDeprecated: true,
+            since: undefined,
+            text: "The Title<br/>some text",
+          },
+          model
+        )
+      ).to.equal("Deprecated. The Title\nsome text");
+    });
+
+    it("replaces link tags with their text", () => {
+      expect(
+        getDeprecationPlainTextSnippet(
+          undefined,
+          {
+            isDeprecated: true,
+            since: undefined,
+            text:
+              "This text has a {@link the_address link with text} and a {@link link_without_text}",
+          },
+          model
+        )
+      ).to.equal(
+        "Deprecated. This text has a link with text and a link_without_text"
+      );
+    });
+  });
+
+  it("only returns the first text sentence when the sentence is followed by a space", () => {
+    expect(
+      getDeprecationPlainTextSnippet(
+        undefined,
+        {
+          isDeprecated: true,
+          since: undefined,
+          text: "This text has two sentences. This is the second sentence.",
+        },
+        model
+      )
+    ).to.equal("Deprecated. This text has two sentences.");
+  });
+
+  it("only returns the first text sentence when the sentence is followed by a newline", () => {
+    expect(
+      getDeprecationPlainTextSnippet(
+        undefined,
+        {
+          isDeprecated: true,
+          since: undefined,
+          text: "This text has two sentences.\nThis is the second line.",
+        },
+        model
+      )
+    ).to.equal("Deprecated. This text has two sentences.");
+  });
+
+  it("only returns the first text sentence when the sentence is followed by a jsdoc <br> tag", () => {
+    expect(
+      getDeprecationPlainTextSnippet(
+        undefined,
+        {
+          isDeprecated: true,
+          since: undefined,
+          text: "This text has two sentences.<br/>This is the second line.",
+        },
+        model
+      )
+    ).to.equal("Deprecated. This text has two sentences.");
+  });
+
+  it("only returns the first 500 characters of the text", () => {
+    expect(
+      getDeprecationPlainTextSnippet(
+        undefined,
+        {
+          isDeprecated: true,
+          since: undefined,
+          text: padEnd(
+            "This text only has one sentence but it is VERY long: ",
+            700,
+            "1234567890"
+          ),
+        },
+        model
+      )
+    ).to.equal(
+      "Deprecated. " +
+        padEnd(
+          "This text only has one sentence but it is VERY long: ",
+          500,
+          "1234567890"
+        ) +
+        "..."
+    );
+  });
+});
+
+describe("The @ui5-language-assistant/logic-utils <convertJSDocToMarkdown> function", () => {
+  const model = buildUI5Model({});
```

Comment:
its best to put `  const model = buildUI5Model({});` in a `before` clause

Even though its fast, it makes debugging any code that is directly inside the `describe`
difficult as even with a (another) test specified as `.only` this initialization code will be invoked on all tests execution and may force defining more complex breakpoints (e.g: stop only after previous breakpoint was already hit).

I wrote like 10,000+ lines of tests without proper `before` clauses in another project and now I am paying the price 😢

Reply:
Done